jcl/src/java.base/share/classes/java/lang/invoke/ArrayVarHandle.java

```diff
@@ -119,9 +119,25 @@
 	}
 /*[ENDIF] Java12 */
 
+/*[IF Java16]*/
+	public MethodType accessModeTypeUncached(AccessType type) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+/*[ELSE]*/
 	public MethodType accessModeTypeUncached(AccessMode accessMode) {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+/*[ENDIF] Java16 */
+
+/*[IF Java16]*/
+	public VarHandle withInvokeExactBehavior() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+
+	public VarHandle withInvokeBehavior() {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+/*[ENDIF] Java16 */
```

Comment:
Similarly can this code block be moved into `java.lang.invoke.VarHandle` as well?